index.html

```diff
@@ -1,16 +1,105 @@
+    <!-- Add a link to your CSS file here (use the line above to guide you) -->
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <header class="header-wrapper">
+        <div class="logo">
+            <img class="logo" src="https://freedesignfile.com/upload/2017/08/Cake-Shop-Logo-Vector.jpg" alt="Logo Image">
+        </div>
+          <div class="btn">
+              <button>
+                Contact
+
+              </button>
+            
+            </div>
+    </header>
+    <section>
+        <nav class = "nav-bar">
+            <ul>
+                <li>HOME</li>
+                <li>CAKE</li>
+                <li>ORDERING</li>
+                <li>LESSONS</li>
+                <li>ABOUT</li>
+                  
+            </ul>
+                
+
+        </nav>
+    </section>
+    <section class="background">
+
+    </section>
+    <section class="container">
+        <div class="row">
+            <div class="col-12 md-col-12">
+                <div>
+                    <h1>
+                        Who we are
+                    </h1>
+                    <p>
+                        The Hummingbird Bakery landed in Notting Hill in early 2004 to become the home of quality American baking in London. Today it has six branches in London and Surrey and is famed the world over for its delicious cupcakes and other freshly baked sweet treats and desserts.
+                    </p>
+
+                </div>
+                    
+            </div>
+            <div class="col-12 md-col-12">
+                <img class="shop" src="https://cdn.shopify.com/s/files/1/1703/7103/files/Islington007_rev1_1000x.jpg?v=1517024508" alt="services Image">
+            </div>
+            <div class="col-6 md-col-3">
+                Child cake
+             <img class="product" src="https://cdn.shopify.com/s/files/1/1703/7103/products/Rainbow_sprikles_pinata_cake_sliced-01_rev0_1000x.jpg?v=1537362594" alt="Child cake">
+            </div>
+            <div class="col-6 md-col-3">
+                Valantin cake
+             <img class="product" src="https://cdn.shopify.com/s/files/1/1703/7103/products/Valentines_2019_cake_3_1000x.jpg?v=1546944659" alt="valantin cake">
+            </div>
+            <div class="col-6 md-col-3">
+                Birthday cake
+             <img class="product" src="https://cdn.shopify.com/s/files/1/1703/7103/products/Vanilla_Mermaid_cake_rev0_large.jpg?v=1573039549" alt="birthday cake">
+            </div>
+            <div class="col-6 md-col-3">
+                Cupcake
+             <img class="product" src="https://cdn.shopify.com/s/files/1/1703/7103/products/Valentines_2019_box_12_minis_rev0_1000x.jpg?v=1544194204" alt="cup cake">
+            </div>
+        </div>
+     </section>  
+    
 </body>
+<footer class="footer">
+    <div class="content">
```

Comment:
You probably don't need this. It wraps the same elements as the footer, and the `content` class doesn't do anything in your CSS.